src/main/java/com/study/HelloWorld/HelloWorldController.java

```diff
@@ -0,0 +1,22 @@
+package com.study.HelloWorld;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+@RestController
+public class HelloWorldController {
+    @RequestMapping("hello")//value属性だけなら省略できる。value = "/hello"
+    String hello() {
```

Comment:
パッケージプライベートでも動くのだと言うのは知りませんでした。
publicつけておくのが無難かとは思います。
```suggestion
    @RequestMapping("hello")//value属性だけなら省略できる。value = "/hello"
    public String hello() {
```